packages/uif/src/BaseIndexer.ts

```diff
@@ -14,6 +14,13 @@ import {
   UpdateEffect,
 } from './reducer/types/IndexerEffect'
 import { IndexerState } from './reducer/types/IndexerState'
+import { Retries, RetryStrategy } from './Retries'
+
+const DEFAULT_RETRY_STRATEGY = Retries.exponentialBackOff({
```

Comment:
Make this configurable, maybe a static field in BaseIndexer

Reply:
5f2b23a